### DISCUSSION TITLE
Let team members log item location

### DISCUSSION THREAD

**Discussion by timvdalen** [Shelf-nu/shelf.nu#294] [General]
After setting up Shelf, I realize I'm an earlier adopter than I thought I was going to be, so I understand if this is still in flux.

Having created an organization and team members so I can give custody of my assets, I don't fully understand the QR code scanning model.
Right now, the admin has to log in to log the location of an item.
Meaning that if the admin signs out an item for a team member to take home, that assets location will never update.

Am I missing something here, or is that the current model?
Is the goal to allow team members to login eventually?

**Comment by DonKoko** [comment]
Hey @timvdalen . Thats a very good question and something we realize is confusing and we are looking into ways to improve.
So the main confusion I think is that we have 2 seperate concepts:

1. Item location - this is meant for where the item is planned/expected to be. So something like **Storage room basement** for example. At the moment this is managed manually and doesn't use any location tracking features or anything like this. Its meant to be manually managed by the asset manager. This will of course improve in the future but we are trying to make it in a simple yet powerful way, so we don't want to rush it. 
2. Scan location - This is the location of the scan. The purpose of this is if you loose an asset and someone scans it you can know where the asset was spotted.

So indeed, we are thinking on how to improve this as we realize its confusing and not the best UX. We are of course open to any suggestions.

**Reply by timvdalen** [reply]
Sorry, I realize I'm mixing up the concepts in my post.

I really only mean 2.
The UX of someone that has custody of an item scanning it to log its scan location is currently confusing to me (because I thought you had to be logged in for it to count, see our other discussion from this morning!).

I think that if a) the bug that causes non-logged-in users not to store location is fixed and b) language is added to the non-authed asset screen to clarify that the scan was logged, my use case is fully solved.

I will add that I'd still be sort of mis-using a few concepts for my usecase: I'd likely rename the user's name & profile image to the org name & logo, because otherwise everything would show up as "Owner" me, instead of the org.

If a user has created a workspace, the Assets page should probably say "{{Workspace name}}'s inventory" rather than "{{user name}}'s inventory". Same with the owner.
